onyo/lib/differs.py

```diff
@@ -37,7 +37,7 @@ def diff_path_change(src: Path, dst: Path) -> Generator[str, None, None]:
 
 def diff_moved_asset(asset_old: dict | Path, asset_new: Path) -> Generator[str, None, None]:
     # could be same. Just check isinstance?
-    yield from diff_path_change(asset_old if isinstance(asset_old, Path) else asset_old.get('path'),
+    yield from diff_path_change(asset_old if isinstance(asset_old, Path) else asset_old.get('onyo.path.absolute'),
```

Comment:
Agree, but I'm not changing that yet, b/c that'll be inconsistent. We currently use absolute paths and they are only sometimes coming from an asset dict. For other operations the paths are the only thing passed around.
So, this can be consistently changed in one go, when we stop passing plain paths, but `Item`s instead.

Reply:
Thanks for the explanation!

Can an explicit TODO be added to note this?